### PR TITLE
replaced the use of fontDescriptorWithSymbolicTraits due to problems in iOS8+ and some fonts

### PR DESCRIPTION
**Pull request by PeterStaev**

### CODE REVIEW

**Comment by enchev** [comment]
Hey @PeterStaev 

Do you think we can have a test to verify if this fix will not break in the future?

Thanks

**Comment by PeterStaev** [comment]
@enchev, 

Added the tests for iOS. Since the problem was present only when you load a custom font, had to add the needed font files as well.

**Comment by enchev** [comment]
@PeterStaev Thanks! Please sign our CLA as well.

**Comment by PeterStaev** [comment]
Already did in the morning when i got the email from @ns-bot.

**Comment by enchev** [comment]
:+1:

File: ui/styling/font.ios.ts [hunk 1]
```diff
@@ -87,10 +87,25 @@ function resolveFontDescriptor(fontFamilyValue: string, symbolicTraits: number):
...
+                fontFaceAttribute = "Regular";
+            } 
+			else {
+				if (symbolicTraits & UIFontDescriptorSymbolicTraits.UIFontDescriptorTraitBold) {
+					fontFaceAttribute += " Bold";
+				}
+				if (symbolicTraits & UIFontDescriptorSymbolicTraits.UIFontDescriptorTraitItalic) {
+					fontFaceAttribute += " Italic";
+				}
+			}
```
Comment (hdeshev) [inline ui/styling/font.ios.ts]: The block above seems to use tabs for indentation. Can you switch to spaces to avoid mixing both spaces and tabs?

**Comment by hdeshev** [comment]
Other than the tabs issue above, LGTM. :+1:

**Comment by vakrilov** [comment]
:+1:

**Comment by dtopuzov** [comment]
run ci

**Comment by nsndeck** [comment]
Thank you for your contribution.
I've committed the code with another pull request (https://github.com/NativeScript/NativeScript/commit/cb92b8864150a15b976672f7ab1be200e2840da8), since code needs some modifications to work. Modifications were necessary in order to be able to use our internal test and build infrastructure. Also some other modifications Including the way the `fontName` value is taken from iOS native view (actually the code of the test does not compile :-)).
Have a nice day.

**Pull request closed without merging**